syntax = "proto3";

package rdk.rpc;


message ServiceConfiguration {
  string friendly_name = 1;
  bool external_host = 2;
  string library_name = 3;
  string service_name = 4;
}

message RpcServerConfiguration {
  int32 num_worker_threads = 1;  
  string listen_address = 2;
  int32 listen_port = 3;
  repeated ServiceConfiguration service = 4;
}
